### ISSUE TITLE
Undefined symbols in v0.13

### ISSUE THREAD

**Issue by christianbioinf** [pysam-developers/pysam#601]
It seems that there is still the undefined symbol for `_main_reheader` as mentioned in #443 since the current checkout of 67b5fe6 causes to the following error for my pyinstaller-onefile script:
```
Traceback (most recent call last):
  File "my_script.py", line 75, in <module>
    from pysam.libcutils import *
  File "/tmp/pip-build-Y5fvcF/pyinstaller/PyInstaller/loader/pyimod03_importers.py", line 396, in load_module
  File "pysam/__init__.py", line 7, in <module>
  File "/tmp/pip-build-Y5fvcF/pyinstaller/PyInstaller/loader/pyimod03_importers.py", line 687, in load_module
ImportError: /tmp/_MEIDQNHTI/pysam.libcbcftools.so: undefined symbol: main_reheader
[31946] Failed to execute script my_script
```
I tracked down the issue. In `bcftools/main.c` there is a declaration of all functions available in the bcftools package including to `main_reheader `but the pysam's bcftools folder does not contain the file `reheader.c` and hence the definition of this function. By simply including this file and its variant `reheader.c.pysam.c`, the issue disappears.

On another note, could you also add the import of `pysam.libctabixproxies` to `pysam/__init__.py` since it is the only one missing there and I always need to manually import it in order to avoid import errors.

**Comment by AndreasHeger** [comment]
Thanks, will sort this out.

**Comment by AndreasHeger** [comment]
Issues seem to be resolved, see #443, closing.
